packages/node_modules/@ciscospark/react-component-avatar/src/styles.css

```diff
@@ -1,23 +1,51 @@
 .avatar {
   display: flex;
-  width: 32px;
-  height: 32px;
+  width: 100%;
+  height: 100%;
   background-size: cover;
   border-radius: 50%;
 }
 
+.avatar.small {
+  width: 18px;
+  height: 18px;
+}
+
+.avatar.medium {
+  width: 24px;
+  height: 24px;
+}
+
+.avatar.large {
+  width: 42px;
+  height: 42px;
+}
+
+
 .avatar-letter {
   display: block;
   width: 100%;
   height: 100%;
   font-size: 20px;
-  line-height: 1.8;
+  line-height: 200%;
   text-align: center;
   background-color: #a9a9a9;
   border-radius: 50%;
 }
 
 .avatar-self {
-  font-size: 34px;
+  font-size: 24px;
   color: #04c9ef;
 }
+
+.small .avatar-letter {
+  font-size: 10px;
```

Comment:
The `font-size` for the avatar letter was the reason I had to go with the class name sizes. It works fine with just the avatar image to set it to fill the space, but not on the fonts.